src/tests/Nav.test.js

```diff
@@ -0,0 +1,13 @@
+import { BrowserRouter } from 'react-router-dom';
+import render from './mocks/testUtils';
+import Nav from '../components/Nav';
+
+test('Home header test', () => {
+  const { getByText, debug } = render(
+    <BrowserRouter>
+      <Nav />
+    </BrowserRouter>,
+  );
+  debug();
+  expect(getByText('Rockets')).toBeInTheDocument();
+});
```

Comment:
- [ ] Good job adding unit test, however, test a test case is failing. Please ensure all test cases are passing!

**Failing test case:**
 ![image](https://user-images.githubusercontent.com/12745474/235334148-d2d821f3-15be-4ed8-b3f3-b6a7f8ebe2c9.png)